### PR TITLE
test jlpm versions, test with nodejs lts, bump build number

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

While we are trying to look into node 10, this will take a look at what we've got today...

### CODE REVIEW

**Comment by jochym** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,12 +30,13 @@ requirements:
     - subprocess32   # [py2k]
 test:
   requires:
-    - nodejs
+    - nodejs >=8,<9
```
[the remark this answers is not in the document]
Comment (jasongrout) [reply recipe/meta.yaml]: (that is, if you want to catch valid node versions. If you explicitly want LTS, never mind.)

**Comment by jasongrout** [comment]
Thanks @bollwyvl! I pulled these changes into #144.

**Comment by bollwyvl** [comment]
Cool!

**Pull request closed without merging**